redfish-client/redfish-client

```diff
@@ -210,6 +214,20 @@ if __name__ == '__main__':
     '''Main application redfish-client'''
     # Functions
 
+    def special_location():
+        '''Check if the installation is in a special location
+        (virtual env, home, etc...)
+
+        :returns: prefix or False
+        :rtype: string or bool
+        '''
+        if(distutils.sysconfig.PREFIX != '/usr' and
+           distutils.sysconfig.PREFIX != '/usr/local'):
+            return(distutils.sysconfig.PREFIX)
+        elif(os.getenv('VIRTUAL_ENV')):
+            return(os.getenv('VIRTUAL_ENV'))
+        return(False)
+
```

Comment:
I don't think this is the role of the code to deal with that, but of an installation script. For me the code should use some env variables to point to the various places which are needed, and then these variables are setup accordingly. But still having hard-coded paths in the program itself seems not the right approach to me.